android/MapsSDKExamplesKotlin/app/src/main/java/com/trimblemaps/mapssdkexampleskotlin/Sample3DBuildingsActivity.kt

```diff
@@ -21,17 +21,6 @@ class Sample3DBuildingsActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        // Authorize the api key for the session.
```

Comment:
These are all just duplicate since the main activity does authorization now?

Reply:
Yes. In the online examples the authorization is included in each example, but in the app we added the menu, so the authorization is performed only once in the SplashActivity